Apps/RaptureIntTests/src/test/java/rapture/lock/LockWorkflowTest.java

```diff
@@ -562,7 +563,9 @@ public void testBlockingLockWithinWorkflows() {
 		       	lockHandle.setLockName(decisionApi.getContextValue(orderUri+"#0", "lockName"));	
 	       	}
         }
-        Assert.assertTrue(lockApi.releaseLock(lockUri.toString(), lockConfig.getName(), lockHandle));
+        try {
```

Comment:
Created this to address lack of documentation:

https://incapture.atlassian.net/browse/RAP-4355